student_explorer/cron.py

```diff
@@ -0,0 +1,168 @@
+# -*- coding: future_fstrings -*-
+
+from __future__ import print_function #python 3 support
+
+from django.db import connections as conns
+
+import logging, datetime, json
+
+from django.conf import settings
+
+from django_cron import CronJobBase, Schedule
+
+from umich_api.api_utils import ApiUtil
+
+from seumich.models import Mentor
+
+import dateutil.parser as parser
+
+from decouple import config
+
+
+logger = logging.getLogger(__name__)
+
+# use ApiUtil for all API directory calls
+apiUtil = ApiUtil(config("APIUTIL_URL", default=""), config("APIUTIL_KEY", default=""), config("APIUTIL_SECRET", default=""))
```

Comment:
These might be better in the settings incase they needed to be used by other modules. We could *probably* even just move the apiUtil here into the settings. Maybe just to keep in mind for the future.

Reply:
These should also be added to the .env.sample file